doc/installation.md

```diff
@@ -37,8 +37,138 @@ bitbag_sylius_inpost_plugin:
 ```
 
 This plugin was made on top
-of [SyliusShippingExportPlugin](https://github.com/BitBagCommerce/SyliusShippingExportPlugin), so please remember to do
-the same for it's configuration.
+of [SyliusShippingExportPlugin](https://github.com/BitBagCommerce/SyliusShippingExportPlugin).
+Typically, Symfony Flex, if you agree, automatically adds the configuration files:
+bitbag_shipping_export_plugin.yaml to the config/packages and config/routes directories.
+It also adding the appropriate entry to config/bundles.php.
+If it doesn't, so please remember to do the same as above for SyliusShippingExportPlugin configuration.
+
+### Create a new controller:
+```php
+// src/Controller/ShippingExportController
+
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use BitBag\SyliusInPostPlugin\Controller\SelectParcelTemplateTrait;
+use BitBag\SyliusShippingExportPlugin\Event\ExportShipmentEvent;
+use BitBag\SyliusShippingExportPlugin\Repository\ShippingExportRepositoryInterface;
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Webmozart\Assert\Assert;
+
+final class ShippingExportController extends ResourceController

```

Comment:
I would extend namespace `BitBag\SyliusShippingExportPlugin\Controller\ShippingExportController` and remove the public methods from the listing.

Reply:
I can't extend ShippingExportController because it is a final class.